extensions/amp-consent/0.1/consent-config.js

```diff
@@ -150,7 +155,52 @@ export class ConsentConfig {
       }
     }
 
-    return config;
+    return this.mergeGeoOverride_(config).then(() => config);
+  }
+
+  /**
+   * Merge correct geoOverride object into toplevel config, then
+   * validate.
+   * @param {JsonObject} config
+   * @return {!Promise}
+   */
+  mergeGeoOverride_(config) {
+    if (config['geoOverride']) {
+      userAssert(
+        config['geoOverride']['geoGroupUnknown'],
+        'geoGroupUnknown must be specified in geoOverride'
```

Comment:
this is actually not mandatory. for example, they could have included `unknown` in one of the `geoGroup` to share settings.

Reply:
Ahh, I misinterpreted the spec.